### DISCUSSION TITLE
Which version of Chipmunk?

### DISCUSSION THREAD

**Discussion by setanarut** [jakecoffman/cp#25] [Q&A] [answered]
We have to know the version to read Chipmunk official documentation. Comment lines are missing in this package. Which version of Chipmunk?

**Comment by jakecoffman** [comment, marked as answer]
I believe it is 7.0.1 which was the latest version at the time I started ported it in 2017.

**Reply by setanarut** [reply]
Thanks, Additionally, all comment lines for this package are missing. This is so annoying. Would you consider adding doc comments and publishing it with a version tag? I can help. If you're interested, I can send a pull request.

**Reply by jakecoffman** [reply]
I don't have much free time lately, PRs would be welcome!
